openscenario/openscenario_interpreter/include/openscenario_interpreter/syntax/double.hpp

```diff
@@ -15,16 +15,17 @@
 #ifndef OPENSCENARIO_INTERPRETER__SYNTAX__DOUBLE_HPP_
 #define OPENSCENARIO_INTERPRETER__SYNTAX__DOUBLE_HPP_
 
-#include <std_msgs/msg/float64.hpp>
 #include <string>
 
 namespace openscenario_interpreter
 {
 inline namespace syntax
 {
-struct Double : public std_msgs::msg::Float64
+struct Double
 {
-  using value_type = decltype(std_msgs::msg::Float64::data);
+  using value_type = double;
+
+  value_type data = 0.0;
```

Comment:
From N4659 11.6 (6.1)

> To zero-initialize an object or reference of type T means:
> if T is a scalar type (6.9), the object is initialized to the value obtained by converting the integer literal
0 (zero) to T

therefore the following code is sufficient:

```suggestion
  value_type data;
```